src/template/context.js

```diff
@@ -16,7 +16,11 @@ const context = {
         },
     },
     regexp: {
-        defaultValue: '\\[default: (.*?)\\]',
+        defaultValue: [
+            '\\[default: (.*?)\\]',
+            '\\(default: (.*?)\\)',
+            '[\\s]+default:[\\s]+(.*?)([\\s]+|$)',
```

Comment:
I think, we can to combine them. Then It will not be necessary to change `setDescription ` function

Reply:
@svoboda-rabstvo done